recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -42,26 +43,62 @@ def update_global_config(feedstock_root):
             _global_config[k1][k2] = repo_config[k1][k2]
 
 
+def fail_if_outdated_windows_ci(feedstock_root):
+    if sys.platform != "win32":
+        return
+
+    if "APPVEYOR_ACCOUNT_NAME" in os.environ:
+        provider = "appveyor"
+        if os.environ["APPVEYOR_ACCOUNT_NAME"] != "conda-forge":
```

Comment:
This is sort of a weird condition. What is it protecting against?

Reply:
> What is it protecting against?

Breaking other organizations using conda-smithy.